nmodule/implmodbus.go

```diff
@@ -0,0 +1,35 @@
+package nmodule
+
+import (
+	"encoding/json"
+	"github.com/NubeIO/lib-module-go/nhttp"
+	"github.com/NubeIO/nubeio-rubix-lib-models-go/dto"
+)
+
+func (g *GRPCMarshaller) WriteValueManually(body *dto.ManualPointWriteValue, opts ...*Opts) (*dto.ManualPointWriteValueResponse, error) {
```

Comment:
Better to have `Modbus` in the prefix. Let's say `ModbusWriteValueManually` otherwise it would be hard to detect the function by the third-party module developers.